AK/Time.h

```diff
@@ -130,26 +130,28 @@ constexpr i64 seconds_since_epoch_to_year(i64 seconds)
     return 1970 + round_down(years_since_epoch);
 }
 
-/*
- * Represents a time amount in a "safe" way.
- * Minimum: 0 seconds, 0 nanoseconds
- * Maximum: 2**63-1 seconds, 999'999'999 nanoseconds
- * If any operation (e.g. 'from_timeval' or operator-) would over- or underflow, the closest legal value is returned instead.
- * Inputs (e.g. to 'from_timespec') are allowed to be in non-normal form (e.g. "1 second, 2'012'345'678 nanoseconds" or "1 second, -2 microseconds").
- * Outputs (e.g. from 'to_timeval') are always in normal form.
- */
-class Time {
+// Represents a time amount in a "safe" way.
+// Minimum: 0 seconds, 0 nanoseconds
```

Comment:
```suggestion
// Minimum: -(2**63) seconds, 0 nanoseconds
```

I realize that that this was already wrong before(?), but let's get possible confusion out of the way as early as possible.